setup.py

```diff
@@ -281,7 +281,13 @@ def _get_manifest(self, build_dir):
         return manifest
 
     def _build_monoclr(self):
-        mono_libs = _check_output("pkg-config --libs mono-2", shell=True)
+        try:
```

Comment:
I think the mono cflags check should be in the try block as well - it also throws in the same way, I believe

Reply:
@dmitriyse is this not failing in CI because both Mono and .NET Core are available?

Reply:
Yes, --xplat was working in CI only due to both frameworks installed.